src/hooks/validation/hook.tsx

```diff
@@ -0,0 +1,27 @@
+import { container } from "tsyringe";
```

Comment:
I have problem with this file and "hooks". Because we have a `ValidationService` which is registered to DI system, and separate hook for validation. Do you want make separation of concerns? We should move code from this file to ValidationService and then use only simple injection. Otherwise, we can remove ValidationService, because it's only a simple wrapper for our `parser-js` and use in this hook `parser-js` API directly. What's your thoughts?

Reply:
I define this as a Hexagonal Architecture, as you say, for separation of concerns to make for example, the hook to only validate the input data type, and the validation service as a wrapper of the parser-js to make it more resilient to future changes. I know it can be a little overkill right now, but thinking on the future of the project, I think that it's the best 🤔

Reply:
Ok, I understand :)